sequencing/ATACSeq/jobSubmission/2_batchCalcQCMetrics.sh

```diff
@@ -6,44 +6,64 @@
 #SBATCH --nodes=1 # specify number of nodes.
 #SBATCH --ntasks-per-node=16 # specify number of processors per node
 #SBATCH --mail-type=END # send email at job completion 
-#SBATCH --output=ATACSeq/logFiles/%u/calcATACQC-%A_%a.o
-#SBATCH --error=ATACSeq/logFiles/%u/calcATACQC-%A_%a.e
-#SBATCH --job-name=calcATACQC
+#SBATCH --output=ATACcalcQCS2-%A_%a.log
```

Comment:
Can you specify in the README, where the log files are located.

Reply:
I'm pretty sure these log files work the same way now as the DNAm pipeline (though they are for an array here instead of a single job). You can likely just adapt what Eilis has already written for that pipeline here (this will make the repository more consistent).

Eilis' explanation below:

>During execution log files are temporaily located in the folder you submitted the script from with the filenames QCDNAdata_XXX.log and QCDNAdata_XXX.err, where XXX is replaced with the job number provided by your HPC scheduler. Please ensure you have the correct permissions to create files in this folder, otherwise the job will instantly fail. If the job fails before the script is run to completion, this is where you need to look for troubleshooting. Once the QC pipeline is complete these files are moved to your data directory in a folder called `logFiles`.